geometry_msgs/Pose pose_for_update
# objects searched
string[] object_type_name_list
---
int32 object_normals
uint32 deactivated_object_normals